app/statistics/server/lib/statistics.js

```diff
@@ -177,14 +178,71 @@ export const statistics = {
 		statistics.onHoldEnabled = settings.get('Livechat_allow_manual_on_hold');
 
 		// Number of Email Inboxes
-		statistics.emailInboxes = await EmailInbox.col.count();
+		statsPms.push(
+			EmailInbox.col.count().then((count) => {
+				statistics.emailInboxes = count;
+			}),
+		);
 
-		statistics.BusinessHours = {
-			// Number of Business Hours
-			total: await LivechatBusinessHours.col.count(),
-			// Business Hours strategy
-			strategy: settings.get('Livechat_enable_business_hours'),
-		};
+		statsPms.push(
+			LivechatBusinessHours.col.count().then((count) => {
+				statistics.BusinessHours = {
+					// Number of Business Hours
+					total: count,
+					// Business Hours strategy
+					strategy: settings.get('Livechat_enable_business_hours'),
+				};
+			}),
+		);
+
+		// Last-Chatted Agent Preferred (enabled/disabled)
+		statistics.lastChattedAgentPreferred = settings.get('Livechat_last_chatted_agent_routing');
+
+		// Assign new conversations to the contact manager (enabled/disabled)
+		statistics.assignNewConversationsToContactManager = settings.get('Omnichannel_contact_manager_routing');
+
+		// How to handle Visitor Abandonment setting
+		statistics.visitorAbandonment = settings.get('Livechat_abandoned_rooms_action');
+
+		// Amount of chats placed on hold
+		statsPms.push(
+			MessagesRaw.col
+				.find({ t: 'omnichannel_placed_chat_on_hold' })
+				.count()
+				.then((count) => {
+					statistics.chatsOnHold = count;
+				}),
+		);
+
+		// VoIP Enabled
+		statistics.voipEnabled = settings.get('VoIP_Enabled');
+
+		// Amount of VoIP Calls
+		statsPms.push(
+			RoomsRaw.col
+				.find({ t: 'v' })
+				.count()
+				.then((count) => {
+					statistics.voipCalls = count;
+				}),
+		);
+
+		// Amount of VoIP Extensions connected
+		statsPms.push(
```

Comment:
This would do way more things than needed for a count. You can get how many users have exts associated by querying the users model with:
```
extension: { $exists: true }
```